packages/teams-js/src/public/app.ts

```diff
@@ -534,11 +534,16 @@ export namespace app {
    * @returns Promise that will be fulfilled when initialization has completed, or rejected if the initialization fails or times out
    */
   export function initialize(validMessageOrigins?: string[]): Promise<void> {
-    return runWithTimeout(
-      () => initializeHelper(validMessageOrigins),
-      initializationTimeoutInMs,
-      new Error('SDK initialization timed out.'),
-    );
+    if (typeof window !== 'undefined') {
+      return runWithTimeout(
+        () => initializeHelper(validMessageOrigins),
+        initializationTimeoutInMs,
+        new Error('SDK initialization timed out.'),
+      );
+    } else {
+      console.log('window is undefined, continuing without initialization');
```

Comment:
console.log is not usually how we do logging in this library. If you look a little further down in this file (line 549) you can see `initializeHelperLogger` get initialized and then used. Following a similar pattern will make sure all of our log statements show up in the same way.